CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ Changes are grouped as follows
 - Separate read/write fields on data classes
 
 ## Unreleased
+### Added
+- Support events filtering by ongoing events (events without `end_time` defined)
+- Support events filtering by active timerange of event
+- Support files metadata filterign by `asset_external_ids`
```

Comment:
```suggestion
- Support files metadata filtering by `asset_external_ids`
```

Reply:
will be fixed with https://github.com/cognitedata/cognite-sdk-python/pull/673